qiskit/tools/visualization/_matplotlib.py

```diff
@@ -682,7 +683,11 @@ def _draw_ops(self, verbose=False):
         #
         # adjust window size and draw horizontal lines
         #
-        max_anc = max([q_anchors[ii].get_index() for ii in self._qreg_dict])
+        anchors = [q_anchors[ii].get_index() for ii in self._qreg_dict]
+        if anchors:
+            max_anc = max(anchors)
+        else:
+            max_anc = 0
         n_fold = (max_anc - 1) // self._style.fold
```

Comment:
@mtreinish Can you change this line as follows, to show registers when empty circuit is given?
```
n_fold = max(0, max_anc - 1) // self._style.fold
```

Reply:
Ok, sure. It is updated with https://github.com/Qiskit/qiskit-terra/pull/1866/commits/2a74b13eff3156bc1b4293a03c49d19d3093fead